dev/docker/nginx/nginx.conf

```diff
@@ -56,7 +56,7 @@ server {
 
         try_files $uri $uri/ @rewrites;
 
-        location ~* \.(?:ico|css|js|gif|jpe?g|png|svg|eot|ttf|woff|mp4|m4v|mov|wmv|avi|mpg|ogv|3gp|3g2|flv|webm|aac|m4a|f4a|mp3|ogg|oga)$ {
+        location ~* \.(?:ico|css|js|gif|jpe?g|png|svg|eot|ttf|woff2?|mp4|m4v|mov|wmv|avi|mpg|ogv|3gp|3g2|flv|webm|aac|m4a|f4a|mp3|ogg|oga|webp)$ {
```

Comment:
@dpellenwood  If we're proxying this stuff from a remote server, wouldn't we want both: `woff` and `woff2`?

Reply:
Yep. the question mark makes the `2` optional for that matching pattern. So both `.woff` and `.woff2` would match.

Reply:
hrmm wouldn't that be `woff?2`, can't remember the details of nginx regexes.

Reply:
Thanks! I double-checked. Nginx use the PCRE format for regexes. ;)